migrations/versions/0321_update_postage_constraint_1.py

```diff
@@ -0,0 +1,81 @@
+"""
+
+Revision ID: 0321_update_postage_constraint_1
+Revises: 0320_optimise_notifications
+Create Date: 2020-03-11 12:01:41.533192
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0321_update_postage_constraint_1'
+down_revision = '0320_optimise_notifications'
+
+
+def upgrade():
+    op.drop_constraint('chk_notifications_postage_null', 'notifications')
+    op.drop_constraint('chk_notification_history_postage_null', 'notification_history')
```

Comment:
Let's try running this script on the copy-production-notify-db on staging. Be good to know how long it will take. 
We might also want to drop and create the constraint on notifications at the same time rather than after both drop.

Reply:
I've reordered the lines in this migration so that it drops the notification constraint, then creates the new notification constraint, and lastly drops the notification_history constraint. I'll also try and find a way to test this with copy-production-notify-db on staging.